src/exporter/utils.py

```diff
@@ -30,7 +30,7 @@ def _extract_data(cls, obj, attrs):
 
         for attr in attrs:
```

Comment:
Como você está usando `attrs[attr]` é melhor fazer `for attr in attrs.values()`

Reply:
Ok